Makefile

```diff
@@ -13,16 +14,16 @@ fioctl-static:
 
 fioctl-linux-amd64:
 fioctl-linux-arm64:
-fioctl-linux-armv7:
+fioctl-linux-arm-7:
```

Comment:
I've never seen it noted that way before. Another thing we could do - stop building it. We don't produce a release artifact for it or ever try and test.

Reply:
We don't build this. So I agree to remove this. A user can anyway try to build any `fioctl-foo-bar` thing from the list of xgo-supported platforms.

The `arm-$X` is how the ARM platforms listed in xgo. I have no preferences for this.